internal/envoy/cluster.go

```diff
@@ -106,6 +114,36 @@ func cluster(cluster *dag.Cluster, service *dag.TCPService) *v2.Cluster {
 	return c
 }
 
+func ClusterLoadAssignConfig(cluster string, service *dag.TCPService) *v2.ClusterLoadAssignment {
```

Comment:
Please call this StaticClusterLoadAssignment. It is a bit of a mouthful, but see the suggested comment.